src/tests/ftest/control/dmg_pool_query_ranks.yaml

```diff
@@ -0,0 +1,21 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+timeouts:
+  test_pool_query_ranks_basic: 120
+  test_pool_query_ranks_error: 120
+  test_pool_query_ranks_mgmt: 360
+server_config:
+  name: daos_server
+  servers:
+    0:
+      scm_class: ram
+      scm_mount: /mnt/daos
+      scm_size: 6
+pool:
+  name: daos_server
+  control_method: dmg
+  scm_size: 4GB
+  nvme_size: 16GB
```

Comment:
```suggestion
```
You're using VMs, so don't need this.

Reply:
Copy & Paste => direct way to hell :'(

- [ ] Remove useless NVME configuration

Reply:
I would also suggest that unless there is a need to specify `scm` vs `nvme`, that it should just specify `size: X`

Reply:
- [ ] Use global size of storage